docs/develop/topics/custom_fields.md

```diff
@@ -21,18 +22,15 @@ To satisfy the the above scenario we will implement a new field `experiments` wh
 }
 ```
 
-This will give us the opportunity to add the experiments a record can belong to, and more information about them.
-
-## Implement a new custom field
-
-We will start by defining a new custom field type.
+Let's define a new custom field type.
```

Comment:
Maybe we can add here a note "For convenience, we will define our new custom field type in the `my-site/invenio.cfg` where the application config resides."